### ISSUE TITLE
Sudden crash, world corruption?

### ISSUE THREAD

**Issue by ampillion** [adam9899/Issues#213]
https://paste.dimdev.org/eloziduwob.mccrash

I was just on the way back towards base, when it suddenly crashed. I could not re-open the world without it immediately crashing again, citing 'unknown' as the possible culprit.

I went back and reloaded the most recent backup after that one, and while it briefly loaded, it then immediately crashed and did the same thing.

I had to go back several hours to find one that would load up and let me actually play. No sooner did I start to move off in a direction, crash. First time, I was using one of the davinci airships, reloaded the world only to find that the airship was gone. Started to try using the vampirism bat mode flight, and as soon as I headed in the direction I was trying to go, it crashed again, with me unable to reload.

**Comment by ByteTemplar** [comment]
Looks like you have NBT data corruption, did you try deleting your playerdata?
How about making new world and exploring new areas?

**Comment by ampillion** [comment]
Tried deleting the playerdata, does the exact same thing still.

I hadn't tried a new world yet, mostly because I didn't really want to lose
progress, but I suppose if all my saves are borked anyway, that'd be the
only step left anyway.

**Comment by ampillion** [comment]
Yeah, creating a new world works, I'll poke around at it some and see if there's something specific that caused it. May have been some kind of corruption going on between this + Win10 stuffing my C drive with updates to the point of filling the drive, which I only caught because the auto backup in game was failing.
